tests/e2e-desktop/scripts/test-recycle-bin.ps1

```
@@ -0,0 +1,158 @@
+    } else {
+        Test-Fail "Create test file (not found after write)"
+    }
+} catch {
+    Test-Fail "Create test file ($_)"
+}
+
+# Capture pre-deletion content
+$PreDeleteContent = ""
+try {
+    $PreDeleteContent = Get-Content -Path "$MountPoint\$TestFile" -Raw -ErrorAction SilentlyContinue
+} catch { }
```

PSScriptAnalyzer (1.24.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)